ifj2017/test/loader.py

```diff
@@ -60,9 +60,20 @@ def _load_compact_tests(self, section_dir):
         extensions = tuple(data.get('extensions', ()))
         try:
             for i, test_case in enumerate(data.get('tests', ())):
-                name = test_case.get('name') or '{:03}'.format(i + 1)
-                code = test_case.get('code') or self._load_test_file(section_dir, name, 'code')
+                name = test_case.get('name')
+                code = test_case.get('code')
 
+                if name and code:
+                    TestLogger.log_warning(
+                        "Redefined test {} in file {}.".format(name, path.join(section_dir, 'tests.json'))
+                    )
```

Comment:
Asi bych v tomto případě přeskakoval načtení tohoto konkrétního testu, protože pak se jeden zakrývá a není to úplně transparentní.